src/viewport/index.tsx

```diff
@@ -118,6 +118,8 @@ class Viewport extends React.Component<
 
         this.visGeometry = new VisGeometry(loggerLevel);
         this.props.simulariumController.visData.clearCache();
+        // TODO this is called in the `VisGeometry` constructor. Why is it called
+        //   again here? Removing it generates arcane WebGL warnings - why?
```

Comment:
I tried removing it in the visGeometry constructor and it doesn't give warnings, but I haven't dug into why